web/src/routes/PoisPage.tsx

```diff
@@ -107,15 +108,25 @@ const PoisPage = ({ match, cityModel, location, languages, history }: PropsType)
           title={title}
           formatter={formatter}
           onInternalLinkClick={history.push}>
-          {location.location && <PageDetail identifier={t('location')} information={location.location} />}
+          {location.location && (
+            <PageDetail
+              identifier={t('location')}
+              information={location.location}
+              mapLink={
+                featureLocation ? `${path.substring(0, path.lastIndexOf('/'))}?${mapParam}=${location.id}` : undefined
```

Comment:
Maybe there is a better way like a build-in js-fkt to do that
I think big libs like that we don't need:
https://www.npmjs.com/package/query-string

Reply:
you cuold actually say: `${createPath(POIS_ROUTE, { cityCode, languageCode })}?${mapParam}=${location.id}`